### PR TITLE
server: add wlr_backend_autocreate() trouble-shooting message

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

**Comment by johanmalm** [comment]
Something like this in support of #860 

If we agree the message here, I can add the same to the website getting-started page.

File: src/server.c [hunk 1]
```diff
@@ -183,6 +183,27 @@ server_global_filter(const struct wl_client *client, const struct wl_global *glo
...
+ * exist, in the hope that it will reduce the time required to get labwc running
+ * and prevent some troubleshooting steps.
+ */
+static const char helpful_seat_error_message[] =
+"\n"
+"Some friendly trouble-shooting help\n"
+"===================================\n"
+"\n"
+"If a seat could not be created, this may be caused by lack of permission to the\n"
+"seat and video groups. If you are using a systemd setup, try installing polkit\n"
```
Comment (Consolatis) [inline src/server.c:199]: `seat, input and video groups`. (Strictly speaking potentially also the `tty` group but mentioning just the 3 seems enough).

**Comment by Consolatis** [comment]
Other than the small nitpick above this looks good to me.
Nice idea to add that message!

**Comment by johanmalm** [comment]
Fixed.
Okay. I'll go for the merge then.
I uninstalled `polkit` to try it out for myself... took longer than I thought it would as so much depended on it :smile:

**Pull request merged**